backend/test/integration/README.md

```diff
@@ -5,7 +5,22 @@
 
 ### How to run
 
+The default integration test will test the default Database, MySQL.
+
 1. Configure kubectl to connect to your kfp cluster.
 2. Run the following for all integration tests: `NAMESPACE=<kfp-namespace> ./run_tests_locally.sh`.
 3. Or run the following to select certain tests: `NAMESPACE=<kfp-namespace> ./run_tests_locally.sh -testify.m Job`.
    Reference: https://stackoverflow.com/a/43312451
+
+### Run database tests with PostgreSQL
+
+To run this test, you need to first deploy the PostgreSQL images on your Kubernetes cluster. For how to deploy, 
```

Comment:
Is it in your next step to automate the test? We can rely on manual test.

Reply:
Ideally we should, and also include PostgreSQL set of tests in the test infra.